eos_data_distribution/MDNS/__init__.py

```diff
@@ -42,8 +42,8 @@ def __init__(self):
         self.pretty_name = avahi.ServiceTypeDatabase.ServiceTypeDatabase()
 
     def get_human_type(self, type):
-        if type in self.pretty_name:
-            return self.pretty_name[type]
+        if str(type) in self.pretty_name:
+            return getattr(self.pretty_name, type)
```

Comment:
This seems very wrong to me. You can't `getattr` anything but a `str`, so the `str(type)` doesn't mean anything.